packages/ui/components/ui/tx/view/undelegate.tsx

```diff
@@ -13,10 +13,6 @@ export const UndelegateComponent = ({ value }: { value: Undelegate }) => {
       label='Undelegate'
       visibleContent={
         <ActionDetails>
-          <ActionDetails.Row label='Epoch index'>
-            {value.startEpochIndex.toString()}
-          </ActionDetails.Row>
```

Comment:
This is a little strange to me: the `startEpochIndex` property is deprecated, so we should no longer show it. Ideally, we'd show the unbonding start height instead – but that property doesn't exist on the `Undelegate` proto! So for now, we'll just hide this, and revisit this if/when core adds that property to the `Undelegate` action.

Reply:
can you leave the code here and just style it hidden or comment it out?